app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -215,6 +215,11 @@
                 {{input type=field.type value=(mut (get data.speaker field.fieldIdentifier))
                         id=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required') (concat 'speaker_' field.fieldIdentifier))}}
               {{/if}}
+            {{else if (eq field.fieldIdentifier 'sponsorshipRequired')}}
```

Comment:
Position of placement of field is wrong, Should be put under `select` field type

Reply:
@kushthedude When putting it under `select` type, it is rendering like following:
![Screenshot from 2020-01-14 17-56-42](https://user-images.githubusercontent.com/56407566/72345300-5f309f00-36f9-11ea-81f6-cc1f9dcb4935.png)

Reply:
@kushthedude @iamareebjamal i have just a question, please clear . 
 What is need to specify field type in form to render it, while there is unique identifier for every field.

Reply:
There are already some fields which are rendered using `field.type  = select` like the language one. Ergo, making more select field and then rendering them using select+field.identifier will just give more clarity. There is no issue in using 
``` js
{{if eq (field.identifier 'gender')}}
```
outside the ``select if`` conditional